lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1839,6 +1839,21 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :description => 'Specify a maximum number of custom Insights events to buffer in memory at a time.',
           :dynamic_name => true
         },
+        :'log_sending.enabled' => {
```

Comment:
This and the following value are very likely to change name but hasn't finalized yet so I'll just touch it once when we get the name.

Reply:
Keep in mind that for the final version, we'll also want to add the settings to `newrelic.yml` so that the community is aware of these options.

Reply:
👍  Good call, thanks @kaylareopelle